modules/ROOT/pages/platform/security/encryption-at-rest.adoc

```diff
@@ -0,0 +1,99 @@
+[[aura-reference-security]]
+= Encryption at rest
```

Comment:
```suggestion
= Encryption
```
Unless we also have sections on _Encryption in transit_ and _End-to-end encryption_, what is the need to specify?